src/roblox_api/cframe.rs

```diff
@@ -0,0 +1,72 @@
+use rbx_dom_weak::types::{CFrame as DomCFrame, Matrix3, Vector3};
```

Comment:
Can we rename the type we define in this module instead of renaming this import? I think we should try to be consistent with a naming scheme for the Lua API tables we export.

Reply:
I renamed it to `CFrameUserData` but let me know if you have something better!